### DISCUSSION TITLE
How to add custom response headers when visiting a website for the first time

### DISCUSSION THREAD

**Discussion by mcpanl** [nuxt/nuxt#25091] [Questions]
Since I need to use SharedArrayBuffer([SharedArrayBuffer Document](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer)), but the browser requires specific response headers to be added for normal use, how should I add these two response headers (besides using coi serviceworker. js, what other methods are there)

**Comment by AloisSeckar** [comment]
Inside server event handler you can access the response object and add a new header from within the `event` context like this:

```js
export default defineEventHandler((event) => {
  event.node.res.setHeader('my-header', 'my-value')
  return 'Hello, world'
})
```
